packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -1,11 +1,18 @@
+# Project-level configuration.
 cmake_minimum_required(VERSION 3.10)
 project(runner LANGUAGES CXX)
 
+# The name of the executable created for the application. Change this to change
+# the on-disk name of your application.
 set(BINARY_NAME "{{projectName}}")
+# The unique GTK application identifier for this application.
 set(APPLICATION_ID "{{linuxIdentifier}}")
 
+# Explicitly opt in to modern CMake behaviors to avoid warnings with recent
+# versions of CMake.
 cmake_policy(SET CMP0063 NEW)
 
+# Ensure that bundled libraries are loaded from the correct location.
```

Comment:
Maybe more precisely (as I understand it):
```
# Ensure that bundled libraries are loaded from the lib/ directory relative to the binary.
```

Reply:
Done.